.circleci/config.yml

```diff
@@ -10,7 +10,9 @@ jobs:
                     python3 -m venv venv
                     . venv/bin/activate
                     pip install requests
-                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ test/optimus/pr
+                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ \
+                                                                           test/smartseq2_single_sample_unpaired/pr \
+                                                                           test/optimus/pr
```

Comment:
Can you also add a trigger_test version of this test below? Once you update from master, there will be an example of how to do that.

Reply:
Okay, rebased and added a trigger_test to circleci